precompiles/outposts/stride/IStrideOutpost.sol

```diff
@@ -47,11 +47,13 @@ interface IStrideOutpost {
     /// @dev Emitted on a Redeem transaction.
     /// @param sender The address of the sender.
     /// @param token The token to be un-luquid staked.
+    /// @param evmosReceiver The address of the receiver on the Evmos chain.
     /// @param receiver The bech32-formatted address of the receiver on the Stride chain.
     /// @param amount The amount of tokens to unstake.
-    event Redeem(
+    event RedeemStake(
         address indexed sender,
         address indexed token,
```

Comment:
nit, group sender and receiver, i.e the `token` field should be below the `receiver`